pkg/controller/jobframework/validation.go

```diff
@@ -51,7 +51,8 @@ var (
 		kftraining.SchemeGroupVersion.WithKind(kftraining.PyTorchJobKind).String(),
 		kftraining.SchemeGroupVersion.WithKind(kftraining.XGBoostJobKind).String(),
 		kfmpi.SchemeGroupVersion.WithKind(kfmpi.Kind).String(),
-		rayv1.SchemeGroupVersion.WithKind("RayJob").String())
+		rayv1.SchemeGroupVersion.WithKind("RayJob").String(),
+		corev1.SchemeGroupVersion.WithKind("Pod").String())
```

Comment:
Will it work with pod-groups? Or just with single pod?

Reply:
Just single pods for now. I am working on a follow up PR for Deployments and Stateful Sets. Should have them in soon!

Reply:
Ah, ok. Thanks!